ReactNativeTemplate/ios/Podfile

```diff
@@ -27,6 +27,9 @@ pod 'React', :path => '../node_modules/react-native', :subspecs => [
     'RCTLinkingIOS'
 ]
 
+# NB: get rid of next line once FMDB works with xcode 9
+pod 'FMDB', :git => 'https://github.com/forcedotcom/fmdb', :branch => '2.7.2_xcode9' 
```

Comment:
No actually once FMDB merges, we can drop that line from our pod files
```
pod 'FMDB', :git => 'https://github.com/forcedotcom/fmdb', :branch => '2.7.2_xcode9'
```

And we will use the office FMDB pod instead because of that [line](https://github.com/wmathurin/SalesforceMobileSDK-iOS/blob/372cba2e5f20835bd4df017bd12ae42c7231e38b/SmartStore.podspec#L23) in SmartStore.podspec

The inline compiling will just be for the main repo.